### PR TITLE
Flexible legal basis input styling

### PR DESCRIPTION
**Pull request by allisonking**
Closes https://ethyca.atlassian.net/browse/PROD-1279

### Description Of Changes

| Before | After |
|--------|--------|
| image  | image |

### Code Changes

* [x] Font size tweak
* [x] Padding tweak

### Steps to Confirm

* [x] Visit a system's privacy declarations and observe the new styling

### Pre-Merge Checklist

* [ ] All CI Pipelines Succeeded
* [x] Update `CHANGELOG.md`

### CODE REVIEW

File: clients/admin-ui/src/features/common/form/inputs.tsx [hunk 1]
```diff
@@ -1020,7 +1020,7 @@ export const CustomSwitch = ({
       <FormControl isInvalid={isInvalid} width="full">
         <Box display="flex" alignItems="center" justifyContent="space-between">
           <HStack spacing={1}>
-            <Label htmlFor={props.id || props.name} fontSize="sm" my={0} mr={0}>
+            <Label htmlFor={props.id || props.name} fontSize="xs" my={0} mr={0}>
```
Comment (allisonking) [inline clients/admin-ui/src/features/common/form/inputs.tsx:1023]: I checked and the only fields that use this stacked switch variant are the ones in the privacy declaration form, so this seemed safe to change (the rest of the forms on that page also use `xs`)

Comment (Kelsey-Ethyca) [reply clients/admin-ui/src/features/common/form/inputs.tsx:1023]: This change feels to broad. It looks ok on the privacy notice form. But it also affects the system info form and I'm not sure it makes sense there. 
Screenshot 2023-10-26 at 11 15 02 AM
Screenshot 2023-10-26 at 11 15 52 AM

Comment (Kelsey-Ethyca) [reply clients/admin-ui/src/features/common/form/inputs.tsx:1023]: might I suggest adding fontSize as an optional prop and have the default be small but then pass in xs for this one instance?

Comment (allisonking) [reply clients/admin-ui/src/features/common/form/inputs.tsx:1023]: via Slack, @Kelsey-Ethyca and I decided we need to standardize all the form inputs to 12px (`xs`). here's what that looks like for the system form:

image

And for privacy declaration form:
image

Comment (Kelsey-Ethyca) [reply clients/admin-ui/src/features/common/form/inputs.tsx:1023]: Thank you Allison!

**Comment by Kelsey-Ethyca** [comment]
UAT ✅ 
Code review ✅

**Review by Kelsey-Ethyca — APPROVED** [review]

**Pull request merged**